This is modeled after the upstream commit which removes ureq as a dependency.

diff --git a/Cargo.toml b/Cargo.toml
index 5a787e1..27171c7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -201,26 +201,3 @@ test-external-apis = []
 upgrade = ["cli"]
 vendored-libgit2 = ["git2/vendored-libgit2"]
 vendored-openssl = ["git2/vendored-openssl"]
-
-[target."cfg(any(target_arch = \"x86_64\", target_arch = \"arm\", target_arch = \"x86\", target_arch = \"aarch64\"))".dependencies.ureq]
-version = "2.7.1"
-features = [
-    "tls",
-    "json",
-    "socks",
-    "socks-proxy",
-]
-default-features = false
-
-[target."cfg(not(any(target_arch = \"x86_64\", target_arch = \"arm\", target_arch = \"x86\", target_arch = \"aarch64\")))".dependencies.native-tls]
-version = "^0.2"
-
-[target."cfg(not(any(target_arch = \"x86_64\", target_arch = \"arm\", target_arch = \"x86\", target_arch = \"aarch64\")))".dependencies.ureq]
-version = "2.7.1"
-features = [
-    "native-tls",
-    "json",
-    "socks",
-    "socks-proxy",
-]
-default-features = false
